amy/autoemails/migrations/0018_auto_20210901_0353.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.24 on 2021-09-01 03:53
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('autoemails', '0017_auto_20210901_0226'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='trigger',
+            name='action',
+            field=models.CharField(choices=[('new-instructor', 'Instructor is added to the workshop'), ('new-supporting-instructor', 'Supporting Instructor is added to the workshop'), ('week-after-workshop-completion', '7 days past the end date of an active workshop'), ('self-organised-request-form', 'A new event is created from Self-Organised Request Form'), ('instructors-host-introduction', 'Introduction of instrutors and host (centr. org. workshop)'), ('ask-for-website', 'Website URL is missing'), ('recruit-helpers', 'Recruit helpers'), ('workshop-request-response1', 'Response to Workshop Request 1'), ('workshop-request-response2', 'Response to Workshop Request 2'), ('workshop-request-response3', 'Response to Workshop Request 3'), ('consent-required', 'There is a new or updated term added that the users should consent to'), ('profile-update', 'Reminder to for the user to update the information in their profile.'), ('profile-archival-warning-inactivity', 'Warning for the user to login before their profile will be archived due to inactivity.')], max_length=50, verbose_name='Action'),
```

Comment:
Can you move this migration to 0017 (there's alter field on `trigger.action` too; I think it can be replaced with this one).